lib/clif-backend/src/signal/mod.rs

```diff
@@ -80,7 +79,10 @@ impl RunnableModule for Caller {
 
             match res {
                 Err(err) => {
-                    *error_out = Some(err.0);
+                    // probably makes the most sense to actually do a translation here to a
+                    // a generic type defined in runtime-core
+                    // TODO: figure out _this_ error return story
```

Comment:
Did you mean to do this before sending out the PR, or to commit the TODO?

Reply:
Outdated comment -- already did it! Thanks!